jwst/residual_fringe/residual_fringe_step.py

```diff
@@ -124,9 +152,17 @@ def process(self, input):
             rfc = residual_fringe.ResidualFringeCorrection(model,
                                                            self.residual_fringe_filename,
                                                            self.regions_filename,
+                                                           self.ignore_regions,
                                                            **pars)
             result = rfc.do_correction()
             result.meta.cal_step.residual_fringe = 'COMPLETE'
             self.output_models.append(result)
 
         return self.output_models
+
```

Comment:
I don't think you need this special class just to raise an exception above. Just use a standard 

`raise ValueError("Number of ignore_region_min does not match ignore_region_max")` above.